docs/topics/OBST-vertx-crud-deploy-to-openshift-pipeline.adoc

```diff
@@ -0,0 +1,22 @@
+## Deploy the application to OpenShift using a Pipeline
+
+When deployed with a _pipeline_ the application is built from the sources (contained in a git repository) by a continuous integration server (Jenkins) running in OpenShift.
```

Comment:
are you just highlighting "pipeline" or do you want it to be a literal?

if its a literal, it needs to be `pipeline`